teradatalabs/cdh5-hive-kerberized/Dockerfile

```diff
@@ -74,11 +74,14 @@ ADD files/conf/hiveserver2-site.xml /etc/hive/conf/hiveserver2-site.xml
 
 # CREATE PRESTO PRINCIPAL AND KEYTAB
 RUN /usr/sbin/kadmin.local -q "addprinc -randkey presto-server/presto-master.docker.cluster@LABS.TERADATA.COM"
+RUN /usr/sbin/kadmin.local -q "addprinc -randkey presto-server/presto-worker.docker.cluster@LABS.TERADATA.COM"
```

Comment:
Some of our environments have only single worker. And it is has the host just `presto-worker` without an index. Although we are not going to use this keytab in such environment, i thought i might be useful at some point. So we wouldn't have to rebuild the images again if we wanted to create a new environment with a single worker.